sqld/src/hrana/mod.rs

```diff
@@ -44,6 +48,10 @@ pub async fn serve(
 
     let mut join_set = tokio::task::JoinSet::new();
     loop {
+        if let Some(isl) = server.idle_shutdown_layer.as_ref() {
+            isl.kick();
+        }
+
```

Comment:
would it make sense to do `server.idle_shutdown_layer.as_ref().map(|s| s.kick());` instead?

Reply:
Unfortunately, the uglifier transforms this into

```rust
        conn.server
            .idle_shutdown_layer
            .as_ref()
            .map(|isl| isl.kick());
```

which is both longer and not more readable than the explicit version :(

Reply:
I like the uglifier name :D